src/cascadia/TerminalApp/CascadiaSettingsSerialization.cpp

```diff
@@ -234,11 +249,13 @@ void CascadiaSettings::_SaveAsPackagedApp(const winrt::hstring content)
     auto file = file_async.get();
 
     DataWriter dw = DataWriter();
-
-    // DataWriter will convert UTF-16 string to UTF-8 (expected settings file encoding)
-    dw.UnicodeEncoding(UnicodeEncoding::Utf8);
-
-    dw.WriteString(content);
+    const char* firstChar = content.c_str();
+    const char* lastChar = firstChar + content.size();
+    // Manually cast to a uint8_t*, because the compiler is unhappy with a static_cast here.
+    const uint8_t* firstByte = (const uint8_t*)firstChar;

```

Comment:
what did it think about a `reinterpret_cast`, if they really need to be casted?

Reply:
@adiviness @zadjii-msft  The cast has to be here because you cannot convert 'const char *' to 'const uint8_t *' - so reinterpret_cast makes sense. 
The example below would be cleaner.
I think this will work based on the [docs](https://docs.microsoft.com/en-us/windows/uwp/cpp-and-winrt-apis/std-cpp-data-types#standard-arrays-and-vectors).
```
std::vector<uint8_t> bytes(content.begin(), content.end());
dw.WriteBytes(bytes);
```
I didn't try this but it looks like it would work. 😄

Reply:
@dlong11 this is my favorite answer, so I'm going with this one. Thanks for the help!